packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -902,6 +917,9 @@ export class MgtPerson extends MgtTemplatedComponent {
 
     const graph = provider.graph.forComponent(this);
 
+    // Prepare person props
+    const personProps = [...this.requestedProps, this.line1Property, this.line2Property, this.line3Property];
```

Comment:
`email` is a special property that is not actually a property that can be fetched from the graph (see the `getTextFromPropoerty` method), we include it for convenience for the developer. However, in this case, it is being requested from the graph, and will be compared to the cache which will never have that property, so the person will always be fetched. Ensure that if `email` is part of any of these properties, it is not included in `personProps`.

Reply:
Ah makes sense, Thanks for the explanation!  I applied a basic filter for the email key, since mail is part of the default properties:

03461db